llvm/include/llvm/ExecutionEngine/Orc/LLJIT.h

```diff
@@ -474,6 +487,11 @@ class LLJITBuilderSetters {
     std::unique_ptr<JITType> J(new JITType(impl(), Err));
     if (Err)
       return std::move(Err);
+
+    if (impl().NotifyCreated)
+      if (Error Err = impl().NotifyCreated(*J))
+        return Err;
```

Comment:
I don't know why the premerge CI didn't fail, but everywhere else in this function is `return std::move(Err);`

I made the change in 5d33f7176b00

Reply:
Seems like clang requires it but not gcc (which is used in premerge) nor MSVC.

You should have got a lot of buildbot emails though, didn't you?

Reply:
I got a lot of buildbot emails, but they also came all day for all kinds of unrelated reasons... Anyway, sorry for the mess.